editor/editor_settings.cpp

```diff
@@ -1220,6 +1219,7 @@ void EditorSettings::_remove_deprecated_settings() {
 	erase("network/connection/engine_version_update_mode");
 	erase("run/output/always_open_output_on_play");
 	erase("run/output/always_close_output_on_stop");
+	erase("text_editor/theme/line_spacing");
```

Comment:
We may want to do something like this every time.
```suggestion
	erase("text_editor/theme/line_spacing"); // See GH-106137.
```